configs/config.json.example

```diff
@@ -367,16 +432,10 @@
         "Any pokemon put here directly force to use Berry & Best Ball to capture, to secure the capture rate": {},
         "any": {"catch_above_cp": 1200, "catch_above_iv": 0.9, "logic": "or" },
         "Lapras": {},
-        "Moltres": {},
-        "Zapdos": {},
-        "Articuno": {},
 
         "// S-Tier pokemons (if pokemon can be evolved into tier, list the representative)": {},
-        "Mewtwo": {},
         "Dragonite": {},
         "Snorlax": {},
-        "// Mew evolves to Mewtwo": {},
-        "Mew": {},

```

Comment:
So, this is the only thing holding this PR up now? Bugger. I'll approve it if you change it back, otherwise... someone else will have to make the call.